driver/include/coyote_defs.h

```diff
@@ -336,6 +336,7 @@ extern bool en_hmm;
 #define IOCTL_SHELL_XDMA_STATS _IOR('F', 16, unsigned long)
 #define IOCTL_SHELL_NET_STATS _IOR('F', 17, unsigned long)
 #define IOCTL_SET_NOTIFICATION_PROCESSED _IOR('F', 18, unsigned long)
+#define IOCTL_GET_INTERRUPT_VALUE _IOR('F', 19, unsigned long)
```

Comment:
This general looks good to me --- very minor comment: let's keep the naming consistent, so let's either have:

IOCTL_SET_NOTIFICATION_PROCESSED and IOCTL_GET_NOTIFICATION_VALUE

or IOCTL_SET_USER_INTERRUPT_PROCESSED and IOCTL_GET_USER_INTERRUPT_VALUE